pkg/controller/tuned/tuned_controller.go

```diff
@@ -234,6 +241,28 @@ func syncDaemonSet(r *ReconcileTuned, tuned *tunedv1alpha1.Tuned) error {
 	return nil
 }
 
+func createCustomResource(mgr manager.Manager) error {
+	client := mgr.GetClient()
+	manifestFactory := manifests.NewFactory()
+
+	// Get the default configuration (CR) for Tuned
+	crManifest, err := manifestFactory.TunedCustomResource()
+	if err != nil {
+		return fmt.Errorf("Couldn't build tuned CustomResource: %v", err)
+	}
+	log.Printf("Built CustomResource manifest for %s/%s\n", crManifest.Namespace, crManifest.Name)
+
+	err = client.Create(context.TODO(), crManifest)
```

Comment:
Still want to lead with `Create()`?

Reply:
Yes in this case.  Had issues leading with .Get() in this case.  Prefer code that is tested and works.  Found two other operators that use this too to create the default CR in this way.